### DISCUSSION TITLE
Geometric path

### DISCUSSION THREAD

**Discussion by Yonaskassa** [winder/Universal-G-Code-Sender#2208] [General]
A per the attached image my ugs geometric path is not moving from the nearest to far it moves unnecessary movement of backward and forward.
<img width="363" alt="Issue" src="https://user-images.githubusercontent.com/131940631/235308292-00f48ed7-fa7c-475f-831c-f908aa1ed0be.PNG">

**Comment by breiler** [comment]
Fixed in: #2210
